polymer.externs.js

```diff
@@ -538,6 +538,12 @@ PolymerElement.prototype._logf = function(var_args) {};
  */
 var PolymerDomApi = function() {};
 
+/**
+ * @param {Node} node
+ * @return {boolean}
+ */
+PolymerDomApi.prototype.deepContains = function(node) {};
```

Comment:
for context of why I'm adding this, it broke chrome's closure compilation build:
http://build.chromium.org/p/chromium.fyi/builders/Closure%20Compilation%20Linux/builds/40482/steps/compile/logs/stdio